vhost-device-vsock/src/vsock_conn.rs

```diff
@@ -229,10 +230,12 @@ impl<S: AsRawFd + ReadVolatile + Write + WriteVolatile> VsockConnection<S> {
 
         match pkt.op() {
             VSOCK_OP_RESPONSE => {
-                // Confirmation for a host initiated connection
-                // TODO: Handle stream write error in a better manner
-                let response = format!("OK {}\n", self.peer_port);
-                self.stream.write_all(response.as_bytes()).unwrap();
+                if self.stream.is_hybrid_vsock() {
+                    // Confirmation for a host initiated connection
+                    // TODO: Handle stream write error in a better manner
+                    let response = format!("OK {}\n", self.peer_port);
+                    self.stream.write_all(response.as_bytes()).unwrap();
+                }
```

Comment:
Maybe we can move the `isHybridVsock` trait and its handling in the first commit.
WDYT?

Reply:
Yeah, makes sense. Done.